### ISSUE TITLE
Segmentation in random parts of the background

### ISSUE THREAD

**Issue by jacobmhkim** [SchmollerLab/Cell_ACDC#20]
**Describe the bug**
When I run module 2, I use the CNN and tracker from YeaZ. I set the threshold to 0.3 and keep everything else the same. I end up with randoms bits of the background segmented as cells.

**To Reproduce**
1. Load module 2 using these files: https://drive.google.com/file/d/1kwLAsSfRuds1_XjPNOYUk7Xr5wlpK_4m/view?usp=sharing
2. Run module 2 using YeaZ as the segmentation CNN, and threshold changed to 0.3
3. Use YeaZ for the tracker
4. Run module 3

**Expected behavior**
When I use the same settings on the YeaZ GUI, I don't get these random blobs that get segmented. 

**Screenshots**
![image](https://user-images.githubusercontent.com/10964296/160215595-1b498369-6d8b-4886-b66e-80096d39e54d.png)

**Desktop (please complete the following information):**
 - Python version: 3.9
 - OS: Windows 10
 - Cell-ACDC Version 1.2.4rc6

**Additional context**

**Comment by ElpadoCan** [comment]
Thanks for reporting this! I always thought it was because of yeaz not being trained specifically on our data, but stupid me, I never thought of trying the actual YeaZ-GUI 🤦‍♂️. I will do some tests, thanks! 

PS: Note that if you want you can also change the background of the segmentation mask on the right (or you can hide it entirely) if you want a lighter background. You find these options with right-click on the right image (if no right-click-action buttons are active) or right-click on the left size colour bar.

**Comment by jacobmhkim** [comment]
Thanks for looking into it. Jordan says that this only happens when he
crops the image.

**Comment by ElpadoCan** [comment]
Ok, I think I fixed it. There was a gaussian filter as a preprocessing step that improved the segmentation with other model organisms that we use, but not with yeast. I removed it now since one can test pre-processing filters from the `Image --> Filters` menu in the main GUI.

Anyway, I just released the fix. Activate the `acdc `environment and then run the command `pip install --upgrade cellacdc` to get the latest version. 

Let me know if it works, thanks!
